rust/arrow/Cargo.toml

```diff
@@ -63,85 +62,8 @@ prettyprint = ["prettytable-rs"]
 memory-check = []
 
 [dev-dependencies]
-criterion = "0.3"
```

Comment:
I guess I was imagining something more like adding a 
```
[features]
benches = ["criterion"]
```

So then if one wanted to have the benches built they could run a command like `cargo bench --features benches ` or something

Reply:
Might be interesting to try out something like that out. It will keep the dev flow more similar to what we have now while reducing the compile times when you don't want to run any benchmarks.

@jorgecarleitao would something like this work for you?